go.mod

```diff
@@ -3,14 +3,15 @@ module github.com/bsm/bfs
 require (
 	github.com/bmatcuk/doublestar v1.2.2
 	github.com/golang/protobuf v1.3.2 // indirect
-	github.com/kr/pretty v0.1.0 // indirect
+	github.com/kr/fs v0.1.0
```

Comment:
you need to add a `bfsscp/go.mod`, see the other sub-dirs for reference. this file should remain unchanged